src/portable.rs

```diff
@@ -292,304 +304,423 @@ impl PortableRegistryBuilder {
 
 #[cfg(test)]
 mod tests {
+    use scale::Compact;
+
     use super::*;
+    use crate::ty::TypeDefPrimitive;
     use crate::{build::*, prelude::vec, *};
 
-    // Type IDs generated by `build_registry`.
```

Comment:
yeah ok these are not needed because make_registry

Reply:
Yeah exactly!